src/index.js

```diff
@@ -115,6 +116,11 @@ async function run(params) {
     params.__ow_query = query;
   }
   const url = `${params.__ow_path.substring(1)}?${params.__ow_query || ''}`;
+  const lvls = url.split('.');
+  // first pop removes 1st level domain
+  lvls.pop();
+  // second pop removes 2nd level domain
+  params.lvl2Dom = lvls.pop();
```

Comment:
also:
I would call it `type` or `kind` here....it could be that in the future, several domains map to the same kind.